src/utils/hwManager.js

```diff
@@ -32,20 +36,70 @@ const getAccountsFromDevice = async ({ device: { deviceId }, networkConfig }) =>
  * signSendTransaction - Function.
  * This function is used for sign a send transaction.
  */
-const signSendTransaction = () => {
-  // TODO implement logic for this function
-  signTransaction();
-  throw new Error('not umplemented');
+const signSendTransaction = async (account, data) => {
+  const transactionObject = createSendTX(
+    account.info.LSK.publicKey,
+    data.recipientId,
+    data.amount,
+    data.data,
+  );
+
+  const transaction = {
+    deviceId: account.hwInfo.deviceId,
+    index: account.hwInfo.derivationIndex,
+    tx: transactionObject,
+  };
+
+  try {
+    const signature = await signTransaction(transaction);
+    const signedTransaction = { ...transactionObject, signature };
+    const result = { ...signedTransaction, id: utils.getTransactionId(signedTransaction) };
+    return result;
+  } catch (error) {
+    return new Error(error);
+  }
 };
 
 /**
  * signVoteTransaction - Function.
  * This function is used for sign a vote transaction.
  */
-const signVoteTransaction = () => {
-  // TODO implement logic for this function
-  signTransaction();
-  throw new Error('not umplemented');
+const signVoteTransaction = async (
+  account,
+  votedList,
+  unvotedList,
+) => {
+  const signedTransactions = [];
+  const votesChunks = splitVotesIntoRounds({ votes: [...votedList], unvotes: [...unvotedList] });
+
+  try {
+    for (let i = 0; i < votesChunks.length; i++) {
+      const transactionObject = {
+        ...castVotes(votesChunks[i]),
+        senderPublicKey: account.publicKey,
+        recipientId: account.address,
+      };
+      // eslint-disable-next-line no-await-in-loop
+      const signature = await signTransaction({
+        deviceId: account.hwInfo.deviceId,
+        index: account.hwInfo.derivationIndex,
+        tx: transactionObject,
+      });
+
+      signedTransactions.push({
+        ...transactionObject,
+        signature,
+        id: utils.getTransactionId({ ...transactionObject, signature }),
+      });
+    }
+
+    return signedTransactions;
+  } catch (error) {
+    return new Error(i18next.t(
```

Comment:
```suggestion
    throw new Error(i18next.t(
```
otherwise it treats the error as `signedTransactions` in
https://github.com/LiskHQ/lisk-hub/blob/153b33e282b01462c7a73bc8dd7ea2ea091d8d7b/src/utils/api/delegates.js#L117-L118
 
and user sees 🐛 
![Screenshot 2019-09-17 at 15 48 36](https://user-images.githubusercontent.com/1254342/65047535-e72cb500-d962-11e9-9ee7-d2eca97b3a15.png)